src/main/kotlin/net/maizegenetics/phgv2/cli/CreateMafVcf.kt

```diff
@@ -118,14 +147,23 @@ class CreateMafVcf : CliktCommand(help = "Create gVCF and hVCF from Anchorwave M
 
     /**
      * Simple Wrapper Function to call biokotlin's MAFToGVCF.getVariantContextsfromMAF
+     * The parameters to BioKotlin's (version 0.10) getVariantContextsfromMAF are:
+     * 1. mafFileName - the name of the MAF file
```

Comment:
Should variable names be enclosed in square brackets to create a link in the generated docs?